SQLite.CodeFirst/SqliteConnectionStringParser.cs

```diff
@@ -26,6 +28,11 @@ internal static class SqliteConnectionStringParser
 
         public static string GetDataSource(string connectionString)
         {
+            if (connectionString.ToLower().Contains(DataDirectoryVariable.ToLower()))
+            {
+                var baseDirectory = AppDomain.CurrentDomain.BaseDirectory + @"\";
```

Comment:
I did the work. May you could test and integrate it?

(Code)[https://gist.github.com/msallin/aa95b8123487020a482e]